ci/prow/e2e-hub-spoke-incluster-build

```diff
@@ -59,9 +59,7 @@ oc -n ${OPERATOR_NAMESPACE} wait --for=condition=complete --timeout=1m build/${s
 timeout 1m bash -c 'until oc -n ${CLUSTER_NAME} get manifestwork/mod-example; do sleep 1; done'
 oc get manifestwork/mod-example -n ${CLUSTER_NAME} -o yaml | yq '.spec.workload.manifests'
 
-# Validate that the Module and the ModuleLoader are running and make sure it has no `build`/`sign` sections
+# Validate that the Module has no `build`/`sign` sections
 timeout 1m bash -c 'until oc -n ${OPERATOR_NAMESPACE} get module/mod-example; do sleep 1; done'
-export moduleloader_pod=$(oc -n ${OPERATOR_NAMESPACE} get pods | grep mod-example | grep -v build | grep -v sign | head -n1 | cut -d" " -f1)
-oc -n ${OPERATOR_NAMESPACE} wait --for=condition=Ready --timeout=1m pod/${moduleloader_pod}
```

Comment:
Don't we want to wait for the worker pod instaed?

Reply:
I don't think so. We want to stop relying on implementation details.